lib/PayPal/Core/PayPalHttpConnection.php

```diff
@@ -93,6 +93,7 @@ public function execute($data)
         switch ($this->httpConfig->getMethod()) {
             case 'POST':
                 curl_setopt($ch, CURLOPT_POST, true);
+                break;
```

Comment:
This is not true. The break is missing for a reason, as the all the above cases should be setting below option.